src/app/store/slices/storage/storage.thunks/deleteItemsThunk.ts

```diff
@@ -8,8 +8,12 @@ import { planThunks } from '../../plan';
 
 export const deleteItemsThunk = createAsyncThunk<void, DriveItemData[], { state: RootState }>(
   'storage/deleteItems',
-  async (itemsToDelete: DriveItemData[], { dispatch }) => {
+  async (itemsToDelete: DriveItemData[], { dispatch, getState }) => {
+    const { workspaces, selectedWorkspace } = getState().workspaces;
+    const workspaceUserId = selectedWorkspace?.workspace?.workspaceUserId || workspaces[0]?.workspace?.workspaceUserId;
```

Comment:
I point this out also in case it needs to be changed so that there is no default workspaceUserId without a workspace selected